bdk_chain/src/tx_graph.rs

```diff
@@ -167,38 +170,87 @@ impl TxGraph {
         })
     }
 
-    /// Iterate over all full transactions in the graph
+    /// Iterate over all full transactions in the graph.
     pub fn full_transactions(&self) -> impl Iterator<Item = &Transaction> {
         self.txs.iter().filter_map(|(_, tx)| match tx {
             TxNode::Whole(tx) => Some(tx),
             TxNode::Partial(_) => None,
         })
     }
 
+    /// Iterate over all partial transactions (outputs only) in the graph.
     pub fn partial_transactions(&self) -> impl Iterator<Item = (Txid, &BTreeMap<u32, TxOut>)> {
         self.txs.iter().filter_map(|(txid, tx)| match tx {
             TxNode::Whole(_) => None,
             TxNode::Partial(partial) => Some((*txid, partial)),
         })
     }
 
-    /// Return an iterator of conflicting txids, where the first field of the tuple is the vin of
-    /// the original tx in which the txid conflicts.
-    pub fn conflicting_txids<'g>(
+    /// Creates an iterator that both filters and maps descendants of the given `txid`.
```

Comment:
When I look at the closure it gives me `OutPointSpends` but from the docs I am expecting a `Transaction` given that `Transaction`s are the "descendants".